src/opnsense/mvc/app/library/OPNsense/Backup/Base.php

```diff
@@ -60,8 +69,12 @@ public function encrypt($data, $pass, $tag = 'config.xml')
             $version = trim(shell_exec('opnsense-version -Nv'));
             $result = "---- BEGIN {$tag} ----\n";
             $result .= "Version: {$version}\n";
-            $result .= "Cipher: AES-256-CBC\n";
-            $result .= "Hash: MD5\n\n";
+            if ($encryption_method === "M0") {
+                $result .= "Cipher: AES-256-CBC\n";
```

Comment:
Please don’t remove this old future proof code. It was paced there to easily upgrade security at a later date which is now. ;)

Reply:
So get rid of this completely?